spel/minimal-linux.pkr.hcl

```diff
@@ -980,6 +1021,41 @@ build {
     ]
   }
 
+  # Azure EL8 provisioners
+  provisioner "shell" {
+    environment_vars = [
+      "SPEL_AMIGENBRANCH=${var.amigen8_source_branch}",
+      "SPEL_AMIGENBUILDDEV=/dev/sda",
+      "SPEL_AMIGENCHROOT=/mnt/ec2-root",
+      "SPEL_AMIGENREPOS=${local.amigen8_repo_names}",
+      "SPEL_AMIGENREPOSRC=${local.amigen8_repo_sources}",
+      "SPEL_AMIGEN8SOURCE=${var.amigen8_source_url}",
+      "SPEL_AMIGENSTORLAY=${local.amigen8_storage_layout}",
+      "SPEL_AMIGENVGNAME=VolGroup00",
+      "SPEL_AMIUTILSSOURCE=${var.amigen_amiutils_source_url}",
+      "SPEL_AWSCFNBOOTSTRAP=${var.amigen_aws_cfnbootstrap}",
+      "SPEL_AWSCLIV1SOURCE=${var.amigen_aws_cliv1_source}",
+      "SPEL_AWSCLIV2SOURCE=${var.amigen_aws_cliv2_source}",
+      "SPEL_BOOTLABEL=/boot",
+      "SPEL_BUILDDEPS=lvm2 parted yum-utils unzip git",
+      "SPEL_BUILDNAME=${source.name}",
+      "SPEL_CLOUDPROVIDER=azure",
+      "SPEL_EXTRARPMS=${local.amigen8_extra_rpms}",
+      "SPEL_FIPSDISABLE=${var.amigen_fips_disable}",
+      "SPEL_GRUBTMOUT=${var.amigen_grub_timeout}",
+      "SPEL_HTTP_PROXY=${var.spel_http_proxy}",
+      "SPEL_USEDEFAULTREPOS=${var.amigen_use_default_repos}",
+    ]
+    execute_command = "{{ .Vars }} sudo -E /bin/bash '{{ .Path }}'"
+    only = [
+      "azure-arm.minimal-centos-8-image",
```

Comment:
There is no centos-8 source for azure yet. This is a bit confusing.
```suggestion
```

Reply:
Yeah. Was copy-paystah'ing @nortiz34's provisional code to create a more cleaned-up configuration. I'd blipped the other `…centos-8…` reference, guess I missed this one.